### DISCUSSION TITLE
Where does the name "Caravela" come from?

### DISCUSSION THREAD

**Discussion by tg73** [metalama/Metalama#2] [Q&A] [answered]
Despite the excellent collection of [most anticipated questions](https://blog.postsharp.net/post/announcing-caravela-preview.html#most-anticipated-questions), surely the most obvious question is, "Why the name Caravela?"

**Comment by gfraiteur** [comment, marked as answer]
We were into ship names for skunk projects. One of our aborted projects was named Frigate when we started Caravela. Caravela was the kind of ships that first used by inhabitants of the "old continent" - Europe - to cross the ocean and discover the new America (the vikings reached America a few centuries earlier but they did that by jumping from island to island and not really by crossing the ocean). For us, Caravela is our vehicle to move from the old and well-known world of MSIL to the Roslyn, new and unknown to us.

**Comment by jerviscui** [comment]
Is commercially charged in the future?

**Reply by gfraiteur** [reply]
There will be a free edition and a commercial edition, we haven't determined the licensing details yet.
